src/test/java/org/jpeek/calculus/java/Lcom4Test.java

```diff
@@ -65,18 +64,19 @@ public void createsXmlCalculusWithXpaths() throws IOException {
         ).affirm();
     }
 
-    @Test
+    @ParameterizedTest
     @Disabled
-    public void methodMethodCalls() throws Exception {
+    @CsvFileSource(resources = "/org/jpeek/calculus/java/lcom4-params.csv")
+    public void methodMethodCalls(final String file, final String value) throws Exception {
         final XML result = new Lcom4().node(
             "", new HashMap<>(0), new Skeleton(
-                new FakeBase("MethodMethodCalls")
+                new FakeBase(file)
             ).xml()
         );
         new Assertion<>(
             "Must create LCOM4 value",
             result.xpath("/metric/app/package/class/@value").get(0),
```

Comment:
@HDouss The Cactoos has `ItemAt` scalar, so you could use it like an elegant way:
`new ItemAt(0, result.xpath("/metric/app/package/class/@value"))`
and test scalars